engine/pkg/meta/model/common.go

```diff
@@ -13,7 +13,30 @@
 
 package model
 
-import "fmt"
+import (
+	"fmt"
+
+	"github.com/pingcap/tiflow/engine/model"
+	"github.com/pingcap/tiflow/engine/pkg/tenant"
+)
+
+// ClientType indicates the kvclient type
+type ClientType = int
+
+type (
+	// ProjectID is the alia of tenant.ProjectID
```

Comment:
What are the benefits of using an alias?  Whether the alias can be deleted？

Reply:
Less pkg import for internal pkg